docs/05-concepts/02-models.md

```diff
@@ -1,6 +1,6 @@
 # Working with models
 
-Models are the YAML files used to define serializable classes in Serverpod. They are used to generate Dart code for the server and client, and, if a database table is defined, to generate database code for the server. The yaml-files are located in the `lib/src/models` directory in your server project.
+Models are Yaml files used to define serializable classes in Serverpod. They are used to generate Dart code for the server and client, and, if a database table is defined, to generate database code for the server. Using regular `.yaml` files within `lib/src/models` is supported, but it is recommended to use `.spy.yaml` (.spy stands for "Server Pod Yaml") both within the models dir and when defining a model elsewhere within `lib/src`.
```

Comment:
> and when defining a model elsewhere within `lib/src`

Suggestion: Remove this part as this is currently not a supported use-case. If you define the models elsewhere the code generator will not pick up the files.